lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -329,6 +329,8 @@ defmodule DynamicSupervisor do
     end
   end
 
+  # Ignore dialyzer warning for fallback clause (invalid child spec)
+  @dialyzer {:no_match, validate_child: 1}
```

Comment:
We probably won't accept those annotations. Dialyzer is not a priority for us and I wouldn't want to add annotations just to make Dialyzer happy. :)

Reply:
I understand, just wanted to ignore some of these to have better visibility at actual issues:
- most importantly, dialyzer errors that may bubble up to elixir end users
- dialyzer issues in erlang itself
- dead code that can be cleaned up

Reply:
Perfect, thank you for all these contributions!